apps/web/src/pages/Home/HomePage/components/Featured/Featured.tsx

```diff
@@ -41,32 +44,59 @@ const Featured = () => {
     ),
   };
 
+  // TEMP: Extracted for now
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const digitalAmbition = {
```

Comment:
So this is the old content, right?  Worth commenting it out totally so it gets stripped on minification?  Or just delete it?

Reply:
Okay, yeah I wasn't sure what to do here. This is a temprorary thing. The reason I left it was because we will likely need this again and I didn't want it to be stripped from our `en.json` and `fr.json` files for translations.

Though, we could always just look in git history to get them back if we need them 🤔

Reply:
Yeah, maybe just leave a link to the PR# they got removed in to make it easy to find it again.